package.json

```
@@ -11,11 +13,16 @@
     "build": "tsc",
     "test": "tsc && ava"
   },
-  "repository": "zeit/uid-promise",
-  "license": "MIT",
   "devDependencies": {
-    "@types/node": "11.9.5",
-    "ava": "0.19.1",
-    "typescript": "3.3.3333"
+    "@types/node": "16.6.1",
+    "ava": "3.15.0",
+    "prettier": "2.3.2",
+    "typescript": "4.3.5"
+  },
+  "prettier": {
+    "arrowParens": "always",
+    "singleQuote": true,
+    "tabWidth": 2,
+    "trailingComma": "none"
```

```suggestion
  "prettier": {
    "singleQuote": true,
    "tabWidth": 2,
    "trailingComma": "all"
```



This should follow the [style-guide](https://github.com/vercel/style-guide)

cc @mrmckeb if we want to make this a public

---

A future project that would be good:

1. Updating our top 20 OSS repos to use consistent Prettier / ESLint from style guide
2. Having them all publish ESM